examples/generate-files.sh

```
@@ -545,9 +502,9 @@ popd > /dev/null
 OUTPUT_FILENAME=${OUTPUT_DIR}/${NAME_SUFFIX}${ENV_FILENAME_SUFFIX}.yaml
 mkdir -p "${OUTPUT_DIR}"
 
-${KUSTOMIZE} build overlays/scope > ${OUTPUT_FILENAME} || exit 1
+${KUSTOMIZE} build ${OVERLAY_DIR_SCOPE} > ${OUTPUT_FILENAME} || exit 1
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)